### DISCUSSION TITLE
How to run ipa radiusproxy-add command via this ansible project

### DISCUSSION THREAD

**Discussion by sdurig1** [freeipa/ansible-freeipa#1161] [Q&A]
How would we go about running the "ipa radiusproxy-add" command via this ansible project?  I saw that in "ipahost" there is a section for "auth_ind", but I don't see any way to specify our server or anything.  Right now we are just doing it through a shell comand in our ansible, but ideally we would have a module or something to do that.

**Comment by t-woerner** [comment]
There is no module for radiusproxy management at the moment in ansible-freeipa. The auth_ind settings for the ipahost and other modules are the same as the auth-ind settings of the corresponding command line commands.

It will be needed to create a new module for radiusproxy management.
